### DISCUSSION TITLE
Avoid go installation with golangci-lint.

### DISCUSSION THREAD

**Discussion by antklim** [golangci/golangci-lint#3351] [Q&A] [answered]
Hi golangci-lint team,

Thanks for the awesome tool. I rely on it a lot.

I apologise if this question was answered before. I'm wondering if it's possible to update the `brew` installation process and not install `go` if it's already present. What happens now is:
- I have `go` installed on my machine. I used an installer from https://go.dev/dl/. 
- I install `golangci-lint` using brew. It implicitly installs `go` using brew. As a result, I have two sources of `go` installed on my machine. Eventually, the versions of different `go` start drifting.

Thanks,
Anton

**Comment by ldez** [comment, marked as answer]
Hello,

We don't maintain the brew formula, it's maintained by the brew community.

https://github.com/Homebrew/homebrew-core/blob/master/Formula/golangci-lint.rb

**Reply by antklim** [reply]
Thanks, @ldez. I opened a question in homebrew - [3925](https://github.com/orgs/Homebrew/discussions/3925). Marking the current question as answered.
